### PR TITLE
2.x - Updated phpbench setup.

### PR DESCRIPTION
**Pull request by othercorey**
This is what it looks like now.  Can update the report to be more concise later on.

Run using: `composer benchmark`

```
+----------------+-------------------------+---------+------+-----+------------+----------+----------+----------+----------+---------+--------+--------+
| benchmark      | subject                 | set     | revs | its | mem_peak   | best     | mean     | mode     | worst    | stdev   | rstdev | diff   |
+----------------+-------------------------+---------+------+-----+------------+----------+----------+----------+----------+---------+--------+--------+
| ConstructBench | benchNow                | chronos | 1000 | 5   | 2,483,800b | 1.843μs  | 1.889μs  | 1.874μs  | 1.957μs  | 0.038μs | 2.02%  | 1.20x  |
| ConstructBench | benchNow                | date    | 1000 | 5   | 2,483,800b | 5.920μs  | 6.262μs  | 6.025μs  | 7.227μs  | 0.486μs | 7.77%  | 3.99x  |
| ConstructBench | benchNowTimezone        | chronos | 1000 | 5   | 2,483,816b | 11.371μs | 11.674μs | 11.816μs | 11.963μs | 0.229μs | 1.96%  | 7.44x  |
| ConstructBench | benchNowTimezone        | date    | 1000 | 5   | 2,483,816b | 5.734μs  | 6.058μs  | 5.874μs  | 6.802μs  | 0.388μs | 6.40%  | 3.86x  |
| ConstructBench | benchRelative           | chronos | 1000 | 5   | 2,483,800b | 2.225μs  | 2.299μs  | 2.250μs  | 2.500μs  | 0.101μs | 4.41%  | 1.46x  |
| ConstructBench | benchRelative           | date    | 1000 | 5   | 2,483,800b | 7.753μs  | 7.995μs  | 8.076μs  | 8.167μs  | 0.151μs | 1.89%  | 5.09x  |
| ConstructBench | benchRelativeTimezone   | chronos | 1000 | 5   | 2,483,816b | 11.687μs | 12.088μs | 12.312μs | 12.462μs | 0.323μs | 2.67%  | 7.70x  |
| ConstructBench | benchRelativeTimezone   | date    | 1000 | 5   | 2,483,816b | 7.952μs  | 8.096μs  | 8.043μs  | 8.340μs  | 0.143μs | 1.76%  | 5.16x  |
| ConstructBench | benchFixed              | chronos | 1000 | 5   | 2,483,800b | 2.321μs  | 2.353μs  | 2.338μs  | 2.392μs  | 0.027μs | 1.16%  | 1.50x  |
| ConstructBench | benchFixed              | date    | 1000 | 5   | 2,483,800b | 5.641μs  | 5.736μs  | 5.769μs  | 5.841μs  | 0.075μs | 1.30%  | 3.65x  |
| ConstructBench | benchFixedTimezone      | chronos | 1000 | 5   | 2,483,816b | 11.866μs | 12.226μs | 12.238μs | 12.567μs | 0.225μs | 1.84%  | 7.79x  |
| ConstructBench | benchFixedTimezone      | date    | 1000 | 5   | 2,483,816b | 5.645μs  | 5.800μs  | 5.725μs  | 6.057μs  | 0.147μs | 2.53%  | 3.69x  |
| ConstructBench | benchCreate             | chronos | 1000 | 5   | 2,483,800b | 10.478μs | 10.501μs | 10.487μs | 10.557μs | 0.029μs | 0.28%  | 6.69x  |
| ConstructBench | benchCreate             | date    | 1000 | 5   | 2,483,800b | 14.794μs | 15.065μs | 14.963μs | 15.472μs | 0.234μs | 1.55%  | 9.60x  |
| ConstructBench | benchCreateTimezone     | chronos | 1000 | 5   | 2,483,816b | 20.414μs | 20.999μs | 20.637μs | 21.892μs | 0.571μs | 2.72%  | 13.38x |
| ConstructBench | benchCreateTimezone     | date    | 1000 | 5   | 2,483,816b | 24.656μs | 25.035μs | 24.799μs | 25.525μs | 0.344μs | 1.37%  | 15.95x |
| ConstructBench | benchFromFormat         | chronos | 1000 | 5   | 2,483,800b | 5.694μs  | 5.757μs  | 5.721μs  | 5.820μs  | 0.054μs | 0.93%  | 3.67x  |
| ConstructBench | benchFromFormat         | date    | 1000 | 5   | 2,483,800b | 9.207μs  | 9.325μs  | 9.319μs  | 9.454μs  | 0.092μs | 0.99%  | 5.94x  |
| ConstructBench | benchFromFormatTimezone | chronos | 1000 | 5   | 2,483,816b | 15.427μs | 15.781μs | 15.482μs | 16.393μs | 0.418μs | 2.65%  | 10.05x |
| ConstructBench | benchFromFormatTimezone | date    | 1000 | 5   | 2,483,816b | 18.755μs | 19.085μs | 19.014μs | 19.561μs | 0.299μs | 1.56%  | 12.16x |
| ConstructBench | benchFromTimestamp      | chronos | 1000 | 5   | 2,483,816b | 2.850μs  | 2.899μs  | 2.882μs  | 2.965μs  | 0.040μs | 1.39%  | 1.85x  |
| ConstructBench | benchFromTimestamp      | date    | 1000 | 5   | 2,483,816b | 7.060μs  | 7.208μs  | 7.247μs  | 7.277μs  | 0.079μs | 1.09%  | 4.59x  |
| ConstructBench | benchFromTimestampUTC   | chronos | 1000 | 5   | 2,483,816b | 1.550μs  | 1.570μs  | 1.559μs  | 1.613μs  | 0.022μs | 1.42%  | 1.00x  |
| ConstructBench | benchFromTimestampUTC   | date    | 1000 | 5   | 2,483,816b | 5.345μs  | 5.520μs  | 5.503μs  | 5.692μs  | 0.118μs | 2.14%  | 3.52x  |
+----------------+-------------------------+---------+------+-----+------------+----------+----------+----------+----------+---------+--------+--------+
```

### CODE REVIEW

File: tests/Benchmark/ConstructBench.php [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+declare(strict_types=1);
+
+/**
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @copyright     Copyright (c) Brian Nesbitt <brian@nesbot.com>
```
Comment (ADmad) [inline tests/Benchmark/ConstructBench.php]: Since this is a new file it doesn't need Brian's copyright.

Comment (othercorey) [reply tests/Benchmark/ConstructBench.php]: Do you have an official header to copy somewhere?

**Comment by othercorey** [comment]
Quick glance shows some interesting stats such as `Date::now()` is 3.3x slower than `Chronos::now()` on average.

File: tests/Benchmark/ConstructBench.php (hunk 1 above)
Comment (markstory) [reply tests/Benchmark/ConstructBench.php]: You only need the copyright line from line 10.

**Pull request merged**